src/main/kotlin/pl/allegro/mobile/logic/operators/numeric/DecimalFormatOperation.kt

```diff
@@ -39,26 +39,26 @@ private class DecimalFormatOperatorFactory {
         .add(element)
         .build()
 
+    @Suppress("MaxLineLength")
```

Comment:
Maybe try splitting this method to avoid the warning? E.g. you can extract creation of the `formatString`